src/containers/ConcentricCircles/NodeLayout.js

```diff
@@ -30,7 +30,7 @@ const dropHandlers = compose(
     onDrop: props => (item) => {
       props.updateNode({
         uid: item.meta.uid,
-        [props.layoutVariable]: relativeCoords(props, item),
+        [props.layoutVariable]: { type: 'layout', ...relativeCoords(props, item) },
```

Comment:
I think there might have been some confusion about this part. Basically, I don't want to "pollute" the node properties with variable type info. The ExportData methods shouldn't need additional properties on the nodes. It should just be able to look at the variable name, look up its type in the variable registry, and when it finds that it is a layout variable parse it in the appropriate way.

Reply:
So an object is only a 'layout' object then if it is in the variableRegistry? Or should variables with type "object" be checked for an "x" and a "y"? Or certain wording in the variable name? 

This was an attempt to catch layout variables we were creating for the sociogram interface based on the protocol, but that weren't in the registry. I knew at that point in the code those were layout variables, but I could also just update the variableRegistry for both protocols.

Reply:
Sorry! It wasn't made clear before, `{ type: 'layout' }` is what has been added to the variable registry, layout variables can cross referenced against that:

```
variableRegistry: {
  node:
    person:
      variables: {
        "closeness": {
            "label": "Closeness layout",
            "description": "More detailed description",
            "type": "layout"
          },
        }
     }
  }
}
```